### PR TITLE
add meeting minutes from 2021-03-30 meeting + make a folder

### LINKED ISSUE #737: OpenJS Foundation Cross Project Council Meeting 2021-03-30
## Time

**UTC Tue 30-Mar-2021 18:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 30-Mar-2021 11:00 (11:00 AM) |
| US / Mountain | Tue 30-Mar-2021 12:00 (12:00 PM) |
| US / Central  | Tue 30-Mar-2021 13:00 (01:00 PM) |
| US / Eastern  | Tue 30-Mar-2021 14:00 (02:00 PM) |
| EU / Western  | Tue 30-Mar-2021 19:00 (07:00 PM) |
| EU / Central  | Tue 30-Mar-2021 20:00 (08:00 PM) |
| EU / Eastern  | Tue 30-Mar-2021 21:00 (09:00 PM) |
| Moscow        | Tue 30-Mar-2021 21:00 (09:00 PM) |
| Chennai       | Tue 30-Mar-2021 23:30 (11:30 PM) |
| Hangzhou      | Wed 31-Mar-2021 02:00 (02:00 AM) |
| Tokyo         | Wed 31-Mar-2021 03:00 (03:00 AM) |
| Sydney        | Wed 31-Mar-2021 05:00 (05:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Cross%20Project%20Council+Meeting+2021-03-30&iso=20210330T1800
* or https://www.wolframalpha.com/input/?i=06PM+UTC%2C+Mar+30%2C+2021+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1CwcsBtK7LqXZv-wjbf41-rOwGGA0xKq3nDQ8_2gb0VA/edit>

## Agenda

Extracted from **cross-project-council-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/summit

* Collaborator summit this year ?  [#306](https://github.com/openjs-foundation/summit/issues/306)

### openjs-foundation/cross-project-council

* Private slack channel for Node.js build team machine management [#729](https://github.com/openjs-foundation/cross-project-council/issues/729)
* doc: add travel fund allocation for 2021 [#725](https://github.com/openjs-foundation/cross-project-council/pull/725)
* Rename master branches [#711](https://github.com/openjs-foundation/cross-project-council/issues/711)
* Improving Diversity and Inclusion at the OpenJSF [#699](https://github.com/openjs-foundation/cross-project-council/issues/699)
* Provide implementation guidance for DCO/CLA [#632](https://github.com/openjs-foundation/cross-project-council/issues/632)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `cross-project-council-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: <https://zoom.us/j/920010234>

* For those who just want to watch: <https://livestream.openjsf.org>

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by jorydotcom**
Closes #737 

also makes a directory for the 2021 meeting notes

### CODE REVIEW

File: meetings/2021/2021-03-30.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* Joe Sepi (@joesepi)
+* Jory Burson (@jorydotcom)
+* Sendil Kumarn (@sendilkumarn)
+* Jordan Harband (@ljharb)
+* Anton Molleda (@molant)
+* Tobie Langel (@tobie)
+* Mike Samuel (@mikesamuel)
+* Sara Chipps
+* Rachel Romoff
+* Timmy Willison
```
Comment (joesepi) [inline meetings/2021/2021-03-30.md:19]: Parris joined as well?

**Review by joesepi — APPROVED** [review]
One comment but otherwise, LGTM. Thanks!!

**Review by ljharb — APPROVED** [review]

**Review by SaraJo — APPROVED** [review]
LGTM

**Review by codebytere — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**